definition_files.rst

```diff
@@ -321,9 +321,8 @@ executed so that they are available during the build and configuration process.
 In some circumstances, it may be redundant to build different containers for
 each app with nearly equivalent dependencies. Singularity supports installing
 apps within internal modules based on the concept of `Standard Container
-Integration Format (SCI-F) <https://sci-f.github.io/>`_
-All the apps are handled by Singularity at this point. More information on
-Apps :ref:`here <apps>`.
+Integration Format (SCIF) <https://sci-f.github.io/>`_. More information on
```

Comment:
This is my bad - the name was ultimately the Scientific Filesystem after community discussion.

Reply:
Gotcha - I'll check through to harmonize it. Just to confirm.... SCIF like the text on the site, or SCI-F like it is in the github URL?

Reply:
SCIF like the text! The one with the dash is only because the other GitHub name was taken :p